impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -100,6 +100,22 @@ std::shared_ptr<Texture> AllocatorVK::OnCreateTexture(
     return nullptr;
   }
 
+  vk::ImageViewCreateInfo view_create_info = {};
```

Comment:
Thinking out aloud: Perhaps we should have a convention for these `FooCreateInfo` structs? Something like `FooCreateInfo foo_desc; auto foo = device.createFoo(foo_desc)`? Here, `view_create_info` is somewhat misleading as you are creating an image_view and the create_info bit is redundant.

Just nits.

Reply:
agreed, I will do a follow-up refactor to fix all these instances.